x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/role_mappings_table.tsx

```diff
@@ -136,11 +136,8 @@ export const RoleMappingsTable: React.FC<IRoleMappingsTableProps> = ({
                     </EuiTableRowCell>
                   )}
                   <EuiTableRowCell>
-                    {(!myRole || myRole.ability.invitableRoleTypes().includes(roleType)) && id && (
-                      <Link to={getRoleMappingPath(id)}>Manage</Link>
-                    )}
-                    {(!myRole || myRole.ability.invitableRoleTypes().includes(roleType)) &&
-                      toolTip && <EuiIconTip position="left" content={toolTip.content} />}
+                    {id && <Link to={getRoleMappingPath(id)}>Manage</Link>}
+                    {toolTip && <EuiIconTip position="left" content={toolTip.content} />}
```

Comment:
In talking to @constancecchen we determined that this was not needed in Kibana. these abilities were not ported over and they are only used in standard auth